src/analysis/tax_transfer_funcs/benefits/wohngeld.py

```diff
@@ -63,31 +73,31 @@ def calc_wg_rent(df, tb, hhsize):
     return np.maximum(wgmiete, min_rent)
 
 
-def calc_max_rent_since_2009(tb, hhsize):
+def calc_max_rent_since_2009(tb, hhsize, mietstufe):
     """
     Since 2009 a different formula for the maximal acknowledged rent applies. Now the
     date of the construction is irrelevant.
     """
     # fixed amounts for the households with size 1 to 5
     # afterwards, fix amount for every additional hh member
     if hhsize <= 5:
-        max_rent = tb["wgmax" + str(hhsize) + "p_m"]
+        max_rent = tb["wgmax{}p_m_st{}".format(hhsize, mietstufe)]
     else:
-        max_rent = tb["wgmax5p_m"] + tb["wgmaxplus5_m"] * (hhsize - 5)
+        max_rent = tb["wgmax5p_m_st{}".format(mietstufe)] + tb["wgmaxplus5_m_st{}".format(mietstufe)] * (hhsize - 5)
     return max_rent
 
 
-def calc_max_rent_until_2008(tb, hhsize, cnstyr):
+def calc_max_rent_until_2008(tb, hhsize, cnstyr, mietstufe):
     """ Before 2009, differentiate by construction year of the house and calculate
     the maximal acknowledged rent."""
     cnstyr_dict = {1: "a", 2: "m", 3: "n"}
     key = cnstyr_dict[cnstyr]
     # fixed amounts for the households with size 1 to 5
     # afterwards, fix amount for every additional hh member
     if hhsize <= 5:
-        max_rent = tb["wgmax" + str(hhsize) + "p_" + cnstyr_dict[cnstyr]]
+        max_rent = tb["wgmax{}p_{}_st{}".format(hhsize, key, mietstufe)]
     else:
-        max_rent = tb["wgmax5p_" + key] + tb["wgmaxplus5_" + key] * (hhsize - 5)
+        max_rent = tb["wgmax5p_{}_st{}".format(key, mietstufe)] + tb["wgmaxplus5_".format(key, mietstufe)] * (hhsize - 5)
```

Comment:
tb["wgmaxplus5_".format(key, mietstufe)] is missing any {} for formatting.

Reply:
where is `any()` in the old code? 88 is new to me, I recall agreeing on 99, but fair enough...

Reply:
At some point, we let black's default be. Please make use of the pre-commit hooks, i.e., type `pre-commit install` once and then this should work automatically. A bit more context is [here](https://econ-project-templates.readthedocs.io/en/stable/pre-commit.html).